src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -453,16 +453,16 @@ namespace AppInstaller::Runtime
                 result.Path /= s_PortablePackagesDirectory;
             }
             break;
-        case PathName::PortablePackageMachineRootX64:
+        case PathName::PortableAppMachineRootX64:
             result.Path = Settings::User().Get<Setting::PortableAppMachineRoot>();
             if (result.Path.empty())
             {
-                result.Path = GetKnownFolderPath(FOLDERID_ProgramFilesX64);
+                result.Path = GetKnownFolderPath(FOLDERID_ProgramFiles);

```

Comment:
`FOLDERID_ProgramFilesX64` does not work for x86 version. We will have to resort to using `FOLDERID_ProgramFiles`.

Reply:
When would you be installing an x64 portable on an x86 machine?